### ISSUE TITLE
Remove uses of apply() where covered by tests

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5707]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)| @vperic reported|
|-|-|
|Trac ID|trac#5707|
|Type|task|
|Created|2012-06-11 13:45:01Z|
|Branch|https://github.com/twisted/twisted/tree/apply-removal-5707|

The apply() built-in is deprecated since 2.3 and has been removed in Python 3. The solution is to just call the function directly. It's used in a handful of places in Twisted, this ticket only covers those which are covered by tests.

<details><summary>Searchable metadata</summary>

```
trac-id__5707 5707
type__task task
reporter__vperic vperic
priority__low low
milestone__Python_3_x Python-3.x
branch__branches_apply_removal_5707 branches/apply-removal-5707
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__py3k py3k
time__1339422301000000 1339422301000000
changetime__1339507243000000 1339507243000000
version__None None
owner__vperic vperic

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_1"></a>|@vperic commented|
|-|-|

(In [34528]) Create branch apply-removal-5707

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_2"></a>|@vperic commented|
|-|-|

Submitting for review, see also [Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/apply-removal-5707) (but I don't see how this could cause errors)

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_3"></a>|@itamarst set owner to @vperic|
|-|-|

Looks good, merge.

**Comment by twisted-trac** [comment]
|[<img alt="vperic's avatar" src="https://avatars.githubusercontent.com/u/688281?s=50" width="50" height="50">](https://github.com/vperic)<a name="note_4"></a>|@vperic set status to `closed`|
|-|-|

(In [34534]) Merge apply-removal-5707: Remove uses of apply()

Author: vperic
Reviewer: itamar
Fixes: [#5707](https://github.com/twisted/twisted/issues/5707)

The apply() built-in has been deprecated since 2.3 and removed in Python 3.
Instead, just call the function directly. This covers the uses of apply()
covered by tests.
